test/calendar_spec.rb

```diff
@@ -0,0 +1,82 @@
+require_relative 'test_helper'
+
+
+describe "Manages calendar and it's various duties" do
+  before do 
+    @reservation1 = Reservation.new("2020/05/04", "2020/05/07")
+    @reservation2 = Reservation.new("2020/05/07", "2020/05/09")
+    @reservation3 = Reservation.new("2020/05/06", "2020/05/11")
+
+    @calendar = Calendar.new
+    @calendar.add_to_calendar(@reservation1)
+    @calendar.add_to_calendar(@reservation2)
+    @calendar.add_to_calendar(@reservation3)
+
+    
+    # puts "\n"
+    # puts @calendar.date_store
+    # puts "\n"
+    
+  end
+
+  it "gives correct cost of requested reservation" do
+    reservation1 = @reservation1.details
+    id = reservation1[:id]
+    cost = @calendar.total_cost_reservation(id)
+
+    expect (reservation1[:cost]).must_equal cost
```

Comment:
Putting a space here means `expect` is called without arguments, which will cause your tests to misbehave.
```suggestion
    expect(reservation1[:cost]).must_equal cost
```